### PR TITLE
Fix for issue #423

### PR DESCRIPTION
**Pull request by chidanandpujar**
changed name to vlan_name for VlanTable
name to client_name for BfdSessionTable

### CODE REVIEW

File: lib/jnpr/junos/op/bfd.yml [hunk 1]
```diff
@@ -42,6 +42,6 @@ _BfdSessionClientTable:
 
 _BfdSessionClientView:
   fields:
-    name: client-name
+    client_name: client-name
```
Comment (dineshbaburam91) [inline lib/jnpr/junos/op/bfd.yml:45]: Please validate for user-defined table view and other custom table views.

Comment (dineshbaburam91) [inline lib/jnpr/junos/op/bfd.yml:45]: Please validate for user-defined table view and other custom table views.

**Comment by chidanandpujar** [comment]
```
Custom table  view VlanTalbe test results for VLanTable 
                                                        
~/pyez_inventory# cat vlan_view_user.py 
from __future__ import print_function
from jnpr.junos import Device
from jnpr.junos.factory.factory_loader import FactoryLoader
import yaml

yaml_data="""
---
VlanTable:
  rpc: get-vlan-information
  item: vlan
  key: vlan-tag
  view: VlanView

VlanView:
  fields:
    instance: vlan-instance
    vlan_name: vlan-name
    created: vlan-create-time
    status: vlan-status
    owner: vlan-owner
    tag: vlan-tag
    members: .//vlan-member-interface
"""

dev = Device(host='xx.xx.xx.xx', user='xyz', password=xyz', gather_facts=False)
dev.open()

globals().update(FactoryLoader().load(yaml.safe_load(yaml_data)))
vlans = VlanTable(dev)
vlans.get()
for vlan in vlans:
  print("instance:",vlan.instance)
  print("vlan_name:", vlan.vlan_name)
  print("created:",vlan.created)
  print("status:",vlan.status)
  print("owner:",vlan.owner)
  print("tag:",vlan.tag)
dev.close()

:~/pyez_inventory# python vlan_view_user.py 
instance: 0
vlan_name: default
created: Fri May 24 20:33:44 2019
status: Enabled
owner: static
tag: 0

PyEZ table VlanTable test results.

from jnpr.junos import Device
from jnpr.junos.op.vlan import VlanTable

dev = Device(host='xx.xx.xx.xx', user='xyz', password='xyz', gather_facts=False)
dev.open()

vlan = VlanTable(dev)
vlan.get()
print(vlan.keys())
print(vlan.values())
dev.close()

~/pyez_inventory# python vlan_view.py 
['default']
[[('instance', '0'), ('vlan_name', 'default'), ('created', 'Fri May 24 20:33:44 2019'), ('status', 'Enabled'), ('owner', 'static'), ('tag', '0'), ('members', 'None')]]

```

**Comment by chidanandpujar** [comment]
PyEZ BfdSessionTable and customer table view test results :

```~/pyez_inventory# cat bfdsession_view.py 
from jnpr.junos import Device
from jnpr.junos.op.bfd import *

dev = Device(host='xx.xx.xx.xx', user='xyz', password='xyz', gather_facts=False)
dev.open()

bfd = BfdSessionTable(dev)
bfd.get()
print(bfd.keys())
print(bfd.values())
dev.close()

~/pyez_inventory# python bfdsession_view.py 
['172.16.1.2']
[[('neighbor', '172.16.1.2'), ('state', 'Up'), ('interface', 'ge-0/0/0.0'), ('detection_time', '3.000'), ('transmission_interval', '1.000'), ('adaptive_multiplier', '3'), ('local_diagnostic', 'None'), ('remote_diagnostic', 'None'), ('version', '1'), ('remote_state', 'Up'), ('minimum_asynchronous_interval', '1.000'), ('minimum_slow_interval', '1.000'), ('adaptive_asynchronous_transmission_interval', None), ('adaptive_reception_interval', '1.000'), ('minimum_transmission_interval', '1.000'), ('minimum_reception_interval', '1.000'), ('detection_multiplier', '3'), ('neighbor_minimum_transmission_interval', None), ('neighbor_minimum_reception_interval', None), ('neighbor_session_multiplier', '3'), ('local_discriminator', '16'), ('remote_discriminator', '16'), ('echo_mode_desired', 'disabled'), ('echo_mode_state', 'inactive'), ('no_absorb', None), ('no_refresh', None), ('bfd_client', _BfdSessionClientTable:evoeventtesty: 0 items)]]


~/pyez_inventory# cat bfd_view_user.py 
from __future__ import print_function
from jnpr.junos import Device
from jnpr.junos.factory.factory_loader import FactoryLoader
import yaml

yaml_data="""
---
BfdSessionTable:
  rpc: get-bfd-session-information
  args:
    extensive: True
  item: bfd-session
  key: session-neighbor
  view: BfdSessionView

BfdSessionView:
  fields:
    neighbor: session-neighbor
    state: session-state
    interface: session-interface
    detection_time: session-detection-time
    transmission_interval: session-transmission-interval
    adaptive_multiplier: session-adaptive-multiplier
    local_diagnostic: local-diagnostic
    remote_diagnostic: remote-diagnostic
    version: session-version
    remote_state: remote-state
    minimum_asynchronous_interval: minimum-asynchronous-interval
    minimum_slow_interval: minimum-slow-interval
    adaptive_asynchronous_transmission_interval: adaptive-asynchronous-transmission_interval
    adaptive_reception_interval: adaptive-reception-interval
    minimum_transmission_interval: minimum-transmission-interval
    minimum_reception_interval: minimum-reception-interval
    detection_multiplier: detection-multiplier
    neighbor_minimum_transmission_interval: neighbor-minimum-transmission_interval
    neighbor_minimum_reception_interval: neighbor-minimum-reception_interval
    neighbor_session_multiplier: neighbor-session-multiplier
    local_discriminator: local-discriminator
    remote_discriminator: remote-discriminator
    echo_mode_desired: echo-mode-desired
    echo_mode_state: echo-mode-state
    no_absorb: { no-absorb: True=regex(no-absorb) }
    no_refresh: { no-refresh: True=no-refresh }
    bfd_client: _BfdSessionClientTable

_BfdSessionClientTable:
  item: bfd-client
  view: _BfdSessionClientView

_BfdSessionClientView:
  fields:
    client_name: client-name
    transmission_interval: client-transmission-interval
    reception_interval: client-reception-interval
"""

dev = Device(host='xx.xx.xx.xx', user='xyz', password='xyz', gather_facts=False)
dev.open()

globals().update(FactoryLoader().load(yaml.safe_load(yaml_data)))
bfds = BfdSessionTable(dev)
bfds.get()
print(bfds.items())
dev.close()

~/pyez_inventory# python bfd_view_user.py 
[('172.16.1.2', [('neighbor', '172.16.1.2'), ('state', 'Up'), ('interface', 'ge-0/0/0.0'), ('detection_time', '3.000'), ('transmission_interval', '1.000'), ('adaptive_multiplier', '3'), ('local_diagnostic', 'None'), ('remote_diagnostic', 'None'), ('version', '1'), ('remote_state', 'Up'), ('minimum_asynchronous_interval', '1.000'), ('minimum_slow_interval', '1.000'), ('adaptive_asynchronous_transmission_interval', None), ('adaptive_reception_interval', '1.000'), ('minimum_transmission_interval', '1.000'), ('minimum_reception_interval', '1.000'), ('detection_multiplier', '3'), ('neighbor_minimum_transmission_interval', None), ('neighbor_minimum_reception_interval', None), ('neighbor_session_multiplier', '3'), ('local_discriminator', '16'), ('remote_discriminator', '16'), ('echo_mode_desired', 'disabled'), ('echo_mode_state', 'inactive'), ('no_absorb', None), ('no_refresh', None), ('bfd_client', _BfdSessionClientTable:evoeventtesty: 0 items)])]
(venv) root@nms5-salt-master-b:~/pyez_inventory# 
```

**Review by dineshbaburam91 — APPROVED** [review]

**Pull request merged**